FY2017/2017-01.beancount

```diff
@@ -4,204 +4,204 @@
 2017-01-01 open Income:Payout USD
 
 ; the earlier date in receipt and bank statements is used for all transactions
-2017-01-01  * "DigitialOcean monthly"
+2017-01-01 * "DigitialOcean monthly"
     freshdesk: "https://gratipay.freshdesk.com/helpdesk/tickets/6502"
     statementPnc: "https://github.com/gratipay/logs/blob/master/statements/2017/01/pnc.csv#L16"
     Expenses:Operations               10.00 USD
     Assets:Operations:PNC            -10.00 USD
 
-2017-01-01  * "MaxCDN monthly"
+2017-01-01 * "MaxCDN monthly"
     freshdesk: "https://gratipay.freshdesk.com/helpdesk/tickets/6479"
     statementPnc: "https://github.com/gratipay/logs/blob/master/statements/2017/01/pnc.csv#L17"
     Expenses:Operations                9.95 USD
     Assets:Operations:PNC             -9.95 USD
 
-2017-01-02  * "bounty award for HackerOne"
+2017-01-02 * "bounty award for HackerOne"
     freshdesk: "https://gratipay.freshdesk.com/helpdesk/tickets/6506"
     statementPnc: "https://github.com/gratipay/logs/blob/master/statements/2017/01/pnc.csv#L18"
     Expenses:Operations               12.00 USD
     Assets:Operations:PNC            -12.00 USD
 
-2017-01-03  * "Braintree funding"
+2017-01-03 * "Braintree funding"
     Assets:Escrow:New-Alliance       121.78 USD
     Liabilities:Escrow              -121.78 USD
 
-2017-01-05  * "Freshdesk monthly"
+2017-01-05 * "Freshdesk monthly"
     freshdesk: "https://gratipay.freshdesk.com/helpdesk/tickets/6535"
     statementPnc: "https://github.com/gratipay/logs/blob/master/statements/2017/01/pnc.csv#L15"
     Expenses:Operations               38.00 USD
     Assets:Operations:PNC            -38.00 USD
 
-2017-01-05  * "Postage"
+2017-01-05 * "Postage"
     freshdesk: "https://gratipay.freshdesk.com/helpdesk/tickets/6536"
     statementPnc: "https://github.com/gratipay/logs/blob/master/statements/2017/01/pnc.csv#L14"
     note: "http://gratipay.slackarchive.io/gratipay/-/1483640673.001811/1483653370.001921/1483650056001917/"
     Expenses:Operations               34.15 USD
     Assets:Operations:PNC            -34.15 USD
 
-2017-01-05  * "Payout"
+2017-01-05 * "Payout"
     freshdesk: "https://gratipay.freshdesk.com/helpdesk/tickets/6530"
     statementPnc: "https://github.com/gratipay/logs/blob/master/statements/2017/01/pnc.csv#L13"
     Income:Payout                    -93.57 USD
```

Comment:
This is a little confusing...I thought that it was mass payout which would have been coming out of Liabilities. Maybe the account should have a different name with reference to Paypal. possibly `Income:Funding:Paypal`